thirdeye-ui/src/app/components/subscription-group-view/notification-channels-card/notification-channels-card.utils.tsx

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2022 StarTree Inc
+ *
+ * Licensed under the StarTree Community License (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at http://www.startree.ai/legal/startree-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT * WARRANTIES OF ANY KIND,
+ * either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Icon } from "@iconify/react";
+import { Box } from "@material-ui/core";
+import React from "react";
+import { lightV1 } from "../../../platform/utils";
+import {
+    NotificationSpec,
+    SpecType,
+} from "../../../rest/dto/subscription-group.interfaces";
+import { SubscriptionGroupViewCardProps } from "../../entity-cards/subscription-group-view-card/subscription-group-view-card.interface";
+
+export const subscriptionGroupChannelHeaderMap = {
+    [SpecType.Webhook]: "label.webhook",
+    [SpecType.Slack]: "label.slack",
+    [SpecType.EmailSendgrid]: "label.email",
+} as const;
+
+export const subscriptionGroupChannelIconsMap = {
+    [SpecType.Webhook]: "material-symbols:webhook-rounded",
+    [SpecType.Slack]: "mdi:slack",
+    [SpecType.EmailSendgrid]: "ic:outline-email",
+} as const;
+
+export const getCardProps = (
+    channel: NotificationSpec,
+    t: (v: string) => string
```

Comment:
Use `TFunction` in `import { TFunction } from "react-i18next";` as the type for `t`

Reply:
Updated accordingly